io-pay-portal-fe/src/index.html

```diff
@@ -50,16 +50,22 @@ <h3>Inserisci i dati dell'avviso</h3>
                       Continua
                     </button>
                   </div>
-                  <div class="spinner-border" id="loading"></div>
-                  <div class="alert alert-danger" role="alert" id="error"></div>
+                  <div class="text-center">
+                    <div class="spinner-border" id="loading"></div>
+                    <div
+                      class="alert alert-danger"
+                      role="alert"
+                      id="error"
+                    ></div>
+                  </div>
                 </form>
               </div>
             </div>
           </div>
           <div class="col-md-6" id="stateCard">
             <div class="card rounded-0 shadow">
               <div class="card-body">
-                <div class="card border-light">
+                <div class="card border-light" id="paymentInfo">
```

Comment:
it would be interesting to have different html templates so that it is more readable (e.g. two different html for each card `paymentInfo ` ant the other one)

But I think it needs to investigate , how to do it with `parcel`